rpcs3/rpcs3qt/game_list_frame.cpp

```diff
@@ -677,12 +696,14 @@ void game_list_frame::Refresh(const bool from_drive, const bool scroll_after)
 
 			m_mutex_cat.unlock();
 
+			const std::string app_identifier = game.path.starts_with(dev_flash) ? game.path.substr(game.path.find_last_of(fs::delim) + 1) : game.serial;
+
```

Comment:
this seems irrelevant as game.serial is already the exact same

Reply:
just tested and it definitely is

Reply:
Fixed.